### DISCUSSION TITLE
Gleiche Darstellung von HM-IP und HM Heizungsgruppen

### DISCUSSION THREAD

**Discussion by udochrist** [OpenCCU/OpenCCU#2093] [🇩🇪 Ideen]
Hallo,

 besteht die Möglichkeit die Darstellung von Heizungsgruppen zwischen HM und HM-IP mit einem Patch der GUI anzugleichen?

Die HM-IP Gruppen zeigen Temp, Luftfeuchte, Fensterstatus und Ventilöffnung:
![Screenshot 2022-12-13 at 16 41 03](https://user-images.githubusercontent.com/869637/207378454-94908c7b-f5e6-487b-8942-ea98ef25a7c4.png)

Die HM Gruppen zeigen leider nur die Temp
![Screenshot 2022-12-13 at 16 44 24](https://user-images.githubusercontent.com/869637/207378759-7be721ab-8778-48c4-a05a-1f1c161fc82d.png)

Ich habe leider nicht ganz verstanden an welcher stelle im WebUI-Code die GUI für Gruppen gebaut wird, aber soweit ich es verstanden habe ist zumindest der DP ACTUAL_HUMIDITY auch in den HM-Gruppen vorhanden und könnte angezeigt werden.

Grüße
Udo

**Comment by jp112sdl** [comment]
> Ich habe leider nicht ganz verstanden an welcher stelle im WebUI-Code die GUI für Gruppen gebaut wird,

[/www/rega/esp/controls/heating_control.fn](https://github.com/eq-3/occu/blob/master/WebUI/www/rega/esp/controls/heating_control.fn)

<img width="1164" alt="Bildschirm­foto 2022-12-15 um 22 17 40" src="https://user-images.githubusercontent.com/18190139/207969012-d4a0b808-dd87-450a-8815-766e8ae10e7e.png">

**Reply by udochrist** [reply]
Sorry für die vielleicht doofe Frage @jp112sdl: Bei dir in der HM-Gruppe ist die Luftfeuchtigkeit drin, oder hast du das auf die Schnelle schon umgebaut? eine HM-IP Gruppe ist das ja nicht nach meinem Verständnis.

**Reply by jp112sdl** [reply]
Das ist eine HM-Gruppe und ja, da habe ich das mit der Luftfeuchtigkeit schnell mal mit reingebaut.

**Reply by jens-maus** [reply]
Na dann her mit dem PR bzw. diff dafür :)

**Reply by jp112sdl** [reply]
Würde ich in `0042-WebUI-HeatingControl-BatteryValveState` ergänzen. 
Soll ich den Patch dann umbenennen in `0042-WebUI-HeatingControl-HumidityBatteryValveState` ?

**Reply by jens-maus** [reply]
Gerne!

**Comment by Baxxy13** [comment]
Betrachtet man die WebUI als reine Administrationsoberfläche ist die Anzeige der Luftfeuchtigkeit im HM-Heizgruppen-Gerät eher irrelevant. Trägt ja nichts zur Heizungssteuerung bei.

Bei IP-Heizgruppen sieht das (inzwischen) etwas anders aus da es dort die Hygrostat-Regelung gibt welche ja auf der Luftfeuchtigkeit fußt.

Ich bräuchte es nicht. ;-)

**Reply by jp112sdl** [reply]
> Betrachtet man die WebUI als reine Administrationsoberfläche

...könnte der Bereich "Status und Bedienung" komplett entfallen 🤷‍♂️☺️🤪

**Reply by Baxxy13** [reply]
Auch wieder wahr. :sunglasses:
